usaspending_api/etl/management/commands/load_table_from_delta.py

```diff
@@ -294,15 +318,31 @@ def handle(self, *args, **options):
             spark.stop()
 
         if postgres_table:
-            logger.info(
+            self.logger.info(
                 f"Note: this has merely loaded the data from Delta. For various reasons, we've separated the"
                 f" metadata portion of the table download to a separate script. If not already done so,"
                 f" please run the following additional command to complete the process: "
                 f" 'copy_table_metadata --source-table {postgres_table} --dest-table {temp_table}'."
             )
 
-    @staticmethod
+    def _set_sequence_value(self, seq_name: str, val: Optional[int] = None) -> int:
+        """
+        Used to reset the value of a Postgres sequence. This function should be used for tables that utilize a
+        sequence to help ensure we don't cross the threshold of values for an ID field as we reload a table every day.
+        If the calling functions exits with an exception this will reset the sequence value to its previous next value.
+
+        Returns the previous value use by the sequence.
+        """
+        new_seq_val = val if val else 1
+        self.logger.info(f"Setting the Postgres sequence to {new_seq_val} for: {seq_name}")
+        with db.connection.cursor() as cursor:
+            cursor.execute(f"SELECT last_value FROM {seq_name}")
+            last_value = cursor.fetchone()[0]
+            cursor.execute(f"ALTER SEQUENCE IF EXISTS {seq_name} RESTART WITH {new_seq_val}")
+        return last_value
```

Comment:
Is the plan still to see if we can drop this as technical debt, by dropping the PK column from the table so that no sequence needs to be maintained? 
What are the remaining challenges to doing so? Django having a PK?

Reply:
>Is the plan still to see if we can drop this as technical debt, by dropping the PK column from the table so that no sequence needs to be maintained?
Not entirely sure if that is planned technical debt or not.

>What are the remaining challenges to doing so? Django having a PK?
Most likely it is just that, but we could trick Django into thinking a PK column exists. The problem is that I have no idea what issues that could cause if Django expects something like a PK but doesn't have one.